.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,12 @@
+---
+name: User story
+about: Tell us something new you want to do
+labels: 
```

Comment:
Do we want user to enter type labels here?

Reply:
No, the user doesn't actually see or edit this part; it seems to be metadata used by github on the "new issue" page.  I haven't tried adding labels to the template to see what it does, but I would guess that it adds these labels automatically to the issue.  I don't think we'd do it for user stories, but we could try it on bugs.